modules/cluster-logging-collector-tuning.adoc

```diff
@@ -149,7 +149,7 @@ $ oc get pods -l component=collector -n openshift-logging
 +
 [source,terminal]
 ----
-$ oc extract configmap/fluentd --confirm
+$ oc extract configmap/collector --confirm
```

Comment:
In Logging 5.8, it's configmap/collector-config 
In Logging 5.5-> 5.7 is configmap/collector

Reply:
Added an additional PR for OCP 4.11 (logging 5.7): https://github.com/openshift/openshift-docs/pull/68531